yt_dlp/extractor/la7.py

```diff
@@ -63,7 +63,7 @@ def _real_extract(self, url):
             url = '%s//%s' % (self.http_scheme(), url)
 
         webpage = self._download_webpage(url, video_id)
-        video_path = self._search_regex(r'(/content/.*?).mp4', webpage, 'video_path')
+        video_path = self._search_regex(r'akamaihd\.net/i(?:/,)?(/content/.*?)\.mp4', webpage, 'video_path')
```

Comment:
Also I would avoid matching blank spaces

```suggestion
        video_path = self._search_regex(r'akamaihd\.net/i(?:/,)?(/content/\S*?)\.mp4', webpage, 'video_path')
```

Reply:
you are correct in this case aswell.